x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/quick_create_job.ts

```diff
@@ -37,6 +32,7 @@ import {
 import { createQueries } from '../utils/new_job_utils';
 import { isCompatibleLayer, createDetectors, getJobsItemsFromEmbeddable } from './utils';
 import { VisualizationExtractor } from './visualization_extractor';
+import { getLens } from '../../../util/dependency_cache';
```

Comment:
`getLens` cannot be used inside this file because the dependency cache is only populated if the user has opened ML.
Instead the lens start contract will need to be passed into the `QuickJobCreator` constructor, in the same way `share` is.

Reply:
done